### ISSUE TITLE
Parse error with domain user

### ISSUE THREAD

**Issue by bang-uin** [sshuttle/sshuttle#467]
https://github.com/sshuttle/sshuttle/blob/9d79bb82c53f75f9bcf362aa214b4230931dfec5/sshuttle/ssh.py#L51

If one uses a domain account then the parse function will break.

```
#~ sshuttle -r user@domain.local@<ip>:<port> 192.168.0.0/16

Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.6/sshuttle", line 11, in <module>
    load_entry_point('sshuttle==1.0.1', 'console_scripts', 'sshuttle')()
  File "/usr/lib64/python3.6/site-packages/sshuttle/cmdline.py", line 102, in main
    opt.sudo_pythonpath)
  File "/usr/lib64/python3.6/site-packages/sshuttle/client.py", line 795, in main
    seed_hosts, auto_hosts, auto_nets, daemon, to_nameserver)
  File "/usr/lib64/python3.6/site-packages/sshuttle/client.py", line 453, in _main
    auto_nets=auto_nets))
  File "/usr/lib64/python3.6/site-packages/sshuttle/ssh.py", line 113, in connect
    username, password, port, host = parse_hostport(rhostport)
  File "/usr/lib64/python3.6/site-packages/sshuttle/ssh.py", line 79, in parse_hostport
    username, host = host.split("@")
ValueError: too many values to unpack (expected 2)
```

**Comment by brianmay** [comment]
This was fixed by #460

**Comment by bang-uin** [comment]
Oh, sorry. Didn't saw that issue.
Thanks!
